hbbft_testing/Cargo.toml

```diff
@@ -22,13 +22,13 @@ edition = "2018"
 travis-ci = { repository = "poanetwork/hbbft" }
 
 [dependencies]
-failure = "0.1.5"
+failure = "0.1.6"
 hbbft = { path = ".." }
 integer-sqrt = "0.1.2"
-proptest = "0.9.2"
+proptest = "0.9.4"
 rand = "0.6.5"
 rand_xorshift = "0.1.1"
-threshold_crypto = "0.3.1"
+threshold_crypto = { git = "https://github.com/kigawas/threshold_crypto", branch = "cleanup-and-codec"}
```

Comment:
Let's wait until https://github.com/poanetwork/threshold_crypto/pull/91 is merged, so we don't have to point outside of https://github.com/poanetwork here.

Reply:
I'll remove it later